src/dialects/mssql/index.js

```diff
@@ -32,7 +32,7 @@ function Client_MSSQL(config = {}) {
     min: 1,
     max: 1,
     idleTimeoutMillis: Number.MAX_SAFE_INTEGER,
-    evictionRunIntervalMillis: 0,
```

Comment:
This is wrong. Here we are setting parameters of internal pool of mssql driver so reapInterval is correct.

Reply:
This is still invalid change.

Those parameters are passed to generic-pool (https://github.com/coopernurse/node-pool) that is used inside mssql driver.  That pool has configuration attribute called `evictionRunIntervalMillis`.

https://github.com/tgriesser/knex/blob/b56ff04e610140d9554ec609986831cd7936960a/src/dialects/mssql/index.js#L217

Reply:
After mssql v6 is released that is no longer the case, though - they moved to tarn.js as well

Reply:
Are they also providing a way to get connection without having to go through their pool? If so I don't think I have anything against mssql anymore (well except it always lags a bit behind of tedious, but we lag more 😅 ). IIRC one impressive feature that tedious had was possibility to sent multiple queries though connection without waiting first that result from previous query is returned. That allows to send many small queries to server in transaction without need to always wait for server roundtrip between sending queries. Maybe mssql supports it as well.

Reply:
Anyways... looks like we have to require that everyone uses mssql > 6.0.0?